src/components/VFooter/VFooter.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <footer class="flex flex-col gap-10 px-6 py-10 lg:px-10 xl:flex-row xl:gap-8">
+    <!-- Logo and links -->
+    <div
+      v-if="isContentMode"
+      class="flex flex-col gap-10 md:flex-row md:items-center md:justify-between xl:flex-grow"
+    >
+      <VLink href="/" class="text-dark-charcoal">
+        <VBrand class="text-[18px]" />
+      </VLink>
+      <nav>
+        <ul class="grid grid-cols-2 gap-6 text-sm sm:flex sm:flex-row sm:gap-8">
+          <li v-for="page in allPages" :key="page.id">
+            <VLink
+              class="text-dark-charcoal"
+              :href="page.link"
+              show-external-icon
+              >{{ $t(`navigation.${page.id}`) }}</VLink
+            >
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <!-- Locale chooser and WordPress affiliation graphic -->
+    <div class="flex flex-col justify-between gap-10 sm:flex-row xl:flex-grow">
+      <div
+        class="flex h-10 w-full w-full items-center justify-center rounded-sm border border-dark-charcoal-20 bg-white sm:w-50"
+      >
+        {{ $t('footer.wip') }}
+      </div>
+      <VLink
+        href="https://wordpress.org"
+        class="text-dark-charcoal hover:no-underline"
+      >
+        <i18n
+          tag="div"
+          path="footer.wordpress-affiliation"
+          class="flex h-full flex-row items-center gap-2 text-sm"
+        >
+          <template #wordpress>
```

Comment:
For some reason, on RTL page this is not aligned vertically. Now, in an `/ar/search` page, 'powered by' is above the center line:
<img width="230" alt="Screen Shot 2022-09-19 at 9 06 54 PM" src="https://user-images.githubusercontent.com/15233243/191084594-9a2e646d-e309-46dc-aea5-291d9385b5ae.png">

Reply:
It's aligned correctly for me:
![image](https://user-images.githubusercontent.com/16580576/191140695-3c89596d-ba63-4e76-9871-8127fe4b915c.png)

I believe it just looks odd because in LTR we see the huge (W) before the text so our perception of centering is thrown off which is not possible when reading RTL. It shouldn't be concerning because if the logo is placed alongside non-English text, the slight baseline mis-alignment will not be apparent.